docs/content/theory/meshes_index.rst

```diff
@@ -0,0 +1,62 @@
+.. _meshes_index:
+
+Meshes
+******
+
+A mesh is a numerical grid on which discrete approximations of continuous functions and variables live.
+Meshes can be created to discretize and solve 1D, 2D or 3D problems.
+For a given system of partial differential equations, the mesh is a discrete representation of the domain and its boundaries.
+Here, we demonstrate mesh types supported by the *discretize* package, and where discrete quantities live.
+
+**Contents:**
+
+  - :ref:`Mesh Types<meshes_index_types>`
+  - :ref:`Where Quantities Live <meshes_index_quantities>`
+
+**Tutorials:**
+
+  - :ref:`Mesh Types Overview <sphx_glr_tutorials_mesh_generation_1_mesh_overview.py>`
+  - :ref:`Tensor Meshes <sphx_glr_tutorials_mesh_generation_2_tensor_mesh.py>`
+  - :ref:`Cylindrical Meshes <sphx_glr_tutorials_mesh_generation_3_cylindrical_mesh.py>`
+  - :ref:`Tree Meshes <sphx_glr_tutorials_mesh_generation_4_tree_mesh.py>`
+
+
+.. _meshes_index_types:
+
+Mesh Types
+==========
+
+Mesh types supported by the *discretize* package include:
+
+    - **Tensor Meshes:** A mesh where the grid locations are organized according to tensor products
+    - **Tree Meshes:** A mesh where the dimensions of cells are :math:`2^n` larger than the dimension of the smallest cell size
+    - **Curvilinear Meshes:** A tensor mesh where the axes are curvilinear
+    - **Cylindrical Meshes:** A pseudo-2D mesh for solving 3D problems with perfect symmetry in the radial direction
+
+.. figure:: ../../images/mesh_types.png
```

Comment:
Do you have the original notebook used to generate these figures somewhere? If we ever want / need to make updates, this would save us a lot of time to have archived (even if right now these are in a separate, small repo -- as long as we have them)

Reply:
I have been keeping the scripts used to make figures with the intent of archiving them eventually. But I wanted to get feedback and finalize the figures first.